addons/io_scene_gltf2/__init__.py

```diff
@@ -429,6 +438,11 @@ def execute(self, context):
         export_settings['gltf_binaryfilename'] = os.path.splitext(os.path.basename(
             bpy.path.ensure_ext(self.filepath,self.filename_ext)))[0] + '.bin'
 
+        export_settings['gltf_user_extensions'] = {
```

Comment:
Can't you just do `export_settings['gltf_user_extensions'] = self.user_extensions` here?

Reply:
indeed. Previously I was doing some checks so no all extensions were added. Now we can leave it as you said